pwiz_tools/Skyline/CleanSkyline.bat

```diff
@@ -41,5 +41,9 @@ IF EXIST ..\Shared\MSGraph\obj rmdir /s /q ..\Shared\MSGraph\obj
 IF EXIST ..\Shared\Crawdad\obj rmdir /s /q ..\Shared\Crawdad\obj
 IF EXIST ProtocolBuffers\tmp rmdir /s /q ProtocolBuffers\tmp
 IF EXIST ProtocolBuffers\GeneratedCode\*.cs del /q ProtocolBuffers\GeneratedCode\*.cs
+IF EXIST TestSettings_x64.testsettings del /q TestSettings_x64.testsettings
+IF EXIST TestSettings_x86.testsettings del /q TestSettings_x86.testsettings
+
+git clean -f -d -X Executables/Installer > nul

```

Comment:
I don't think this batch file should use "git clean".
If a developer has files in their source tree that have not yet added to git, we would not expect that those files will be deleted by doing a clean build.

Reply:
I agree it's really bad to delete something that isn't generated by the build. But note that git clean is used by the main pwiz clean scripts to clean up the vendor API and data directories, and the command above that deletes GeneratedCode/*.cs, and the ones that delete entire directories, are equally non-discriminating. But I'd accept that unadded files are more likely to happen in Executables/Installer than GeneratedCode. ;) If you're ok with the MSI being built in a bin directory, I'll switch to that and avoid the cleaning issue entirely.

Reply:
Yes, bin directory would be a great place for the .msi file. That's where it currently gets put in the master branch, right?

Reply:
The README implies it gets build in the Installer directory but I didn't actually test it. The vdproj doesn't load in my Visual Studio. Didn't they disable it in 2017? Are you using 2015 to build it?